lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -131,6 +132,8 @@ def call(params)
               filter = UserBricksHelper.resolve_client_id(domain, project, params.data_product)
             end
             user_filters = user_filters.select { |f| f[:pid] == filter } if filter
+
+            GoodData.gd_logger.info("Synchronizing in mode=#{mode}, project_id=#{project.pid}, data_rows=#{user_filters.nil? ? 0 : user_filters.size}")
```

Comment:
why do we need to check user_filters.nil? here? in above comment you said that it cannot be nil.

Reply:
In above comment, It is already initial [] for user_filters in load_data(...) so It cannot be nil.